File: CursoPOOUber/Java/Driver.java
package Java;

public class Driver extends Account{

    public Driver(String name, String document, String email, String password) {
        super(name, document);
        super.email = email;
        super.password = password;
    }
    
    void printDriverData(){
        System.out.println("Document driver: " + document + " Name driver: " + name + " E-mail: " + email + " Password: " + password);
    }
}

File: CursoPOOUber/JS/Driver.js
class Driver extends Account{
    constructor(name, document, email, password){
        super(name, document, email, password)
    }

    printDataDriver() {
        console.log("Driver Name: " + this.name + " Driver document: " + this.document + " Driver email: " + this.email + " Driver password: " + this.password);
    }
}